pixi.toml

```diff
@@ -12,6 +12,8 @@ channels = ["https://prefix.dev/conda-forge"]
 platforms = ["linux-64", "win-64", "osx-64", "osx-arm64"]
 
 [dependencies]
+mkdocs-material = ">=9.5.49,<10"
```

Comment:
Should these only be added to the docs feature?

Reply:
Yep